packages/wallets/src/evm/walletIds.ts

```diff
@@ -0,0 +1,11 @@
+export const walletIds = {
```

Comment:
There's a `constants` folder in this path, it's probably worth it to move this file in there

Reply:
done